sematic/ui/packages/common/src/ApiContracts.ts

```diff
@@ -1,4 +1,4 @@
-import { Resolution, Run, User } from "src/Models";
+import { Artifact, Edge, Resolution, Run, User } from "src/Models";
```

Comment:
Unrelated to this PR - FYI: When resizing the window to make it smaller, the Notes panel collapse arrow floats and is positioned incorrectly:

![image](https://github.com/sematic-ai/sematic/assets/1894533/327a6acf-9c43-4713-afd6-25ef688e93b5)

Reply:
Noted. Yes, it is unrelated to this PR. I will fix it in the subsequent PR. Looks like I over-deployed :D